<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimMobility Road Network Editor in PyQt4</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_show_selection_box"/>
    <addaction name="action_show_polyline_dots"/>
    <addaction name="action_snap_to_line"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_extract_crossings_and_stop_lines"/>
    <addaction name="action_extract_yellow_box"/>
    <addaction name="action_create_missing_stop_line"/>
    <addaction name="action_extract_road_segments"/>
    <addaction name="action_edit_selected_road_item"/>
    <addaction name="action_delete_lane_edge"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="action_snap_to_line"/>
  </widget>
  <widget class="QDockWidget" name="selection_box">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Selection box</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="selection_box_vertical_layout">
     <item>
      <widget class="QTreeWidget" name="selection_tree_widget">
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>20</number>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>group</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>visible</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>selectable</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="selection_scroll_area">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>254</width>
          <height>248</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="selection_label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_save">
   <property name="icon">
    <iconset resource="pqrne.qrc">
     <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save changes to databases.</string>
   </property>
   <property name="statusTip">
    <string>Save changes to databases.</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="pqrne.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_show_selection_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show selection box</string>
   </property>
   <property name="toolTip">
    <string>Show/hide the selection box.</string>
   </property>
   <property name="statusTip">
    <string>Show/hide the selection box.</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="pqrne.qrc">
     <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a different set of shape-file and road-network databases.</string>
   </property>
   <property name="statusTip">
    <string>Open a different set of shape-file and road-network databases.</string>
   </property>
  </action>
  <action name="action_extract_yellow_box">
   <property name="icon">
    <iconset>
     <normaloff>:/images/lettery.jpg</normaloff>:/images/lettery.jpg</iconset>
   </property>
   <property name="text">
    <string>Extract yellow boxes</string>
   </property>
   <property name="toolTip">
    <string>Extract yellow boxes in the selection region or the entire map.</string>
   </property>
   <property name="statusTip">
    <string>Extract yellow boxes in the selection region or the entire map.</string>
   </property>
  </action>
  <action name="action_extract_crossings_and_stop_lines">
   <property name="text">
    <string>Extract crossings &amp;&amp; stop-lines</string>
   </property>
   <property name="toolTip">
    <string>Extract crossings &amp; stop-lines in the selection region or entire map</string>
   </property>
   <property name="statusTip">
    <string>Extract crossings &amp; stop-lines in the selection region or entire map</string>
   </property>
  </action>
  <action name="action_show_polyline_dots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show polyline dots</string>
   </property>
   <property name="toolTip">
    <string>Show/hide the individual points of the polyline.</string>
   </property>
   <property name="statusTip">
    <string>Show/hide the individual points of the polyline.</string>
   </property>
  </action>
  <action name="action_create_missing_stop_line">
   <property name="text">
    <string>Create missing stop-line</string>
   </property>
   <property name="toolTip">
    <string>Create missing stop-line in the selection region</string>
   </property>
   <property name="statusTip">
    <string>Create missing stop-line in the selection region</string>
   </property>
  </action>
  <action name="action_edit_selected_road_item">
   <property name="text">
    <string>Edit selected road-item</string>
   </property>
   <property name="toolTip">
    <string>Edit selected road-item's attributes such as its type and polyline</string>
   </property>
   <property name="statusTip">
    <string>Edit selected road-item's attributes such as its type and polyline</string>
   </property>
  </action>
  <action name="action_delete_lane_edge">
   <property name="text">
    <string>Delete lane-edge</string>
   </property>
  </action>
  <action name="action_snap_to_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snap to line</string>
   </property>
  </action>
  <action name="action_extract_road_segments">
   <property name="text">
    <string>Extract road segments</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pqrne.qrc"/>
 </resources>
 <connections/>
</ui>
